app/models/cinstance.rb

```diff
@@ -391,6 +392,10 @@ def app_plan_change_should_request_credit_card?
 
   protected
 
+  def account_for_sphinx
```

Comment:
`alias_method` maybe?

Reply:
hmm I want it not to be public but yes why not
Thing is I do not want people to use this outside of this class